### DISCUSSION TITLE
Assign Variable not working

### DISCUSSION THREAD

**Discussion by Ajsingh686** [postmanlabs/postman-flows#276] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/85873688/157373734-f22b7543-13eb-411e-8b7d-22611fa42367.png)

![image](https://user-images.githubusercontent.com/85873688/157374704-890d0749-3413-48f6-9678-13363cb95bcf.png)

My previous request has been successfully ran and the response of this request has a id that i need to use in next request, Now I am trying to assign this id to a variableI, But i am not geting that response value in assign variable, not even in terminal. Please Help,,I have previously created many flows that are working fineand running but when i am creating new flow it is not happening.

Thanks

**Comment by ArminKugler** [comment, marked as answer]
Try running your send request in the collection successfully and the **Save the response**. The Flow should know what  to expect and give  you proposals.

**Reply by Ajsingh686** [reply]
Thanks you man,,,its worked, At first it is not asking for saving the response, it was taken only from flows...
BUt okk,,i have got that..Thanks

**Reply by ArminKugler** [reply]
you are welcome. Also that was a keylearning for myself last week. Actually credits got to Beth Marshall who did very nice youtube videos on postman flows.
